pkg/compiler/test/compiler_test.go

```diff
@@ -109,21 +109,25 @@ func TestBranches(t *testing.T) {
 		}
 
 		t.Run(path, func(t *testing.T) {
-			//if !strings.HasSuffix(path, "success_5.json") {
+			//if !strings.HasSuffix(path, "success_6.json") {
```

Comment:
huh?!

Reply:
what happened to the previous test case?

Reply:
this is just commented out... it's to skip the irrelevant test cases - a test was added so it was bumped.